charts/backbeat/values.yaml

```diff
@@ -1,12 +1,25 @@
 # Default values for backbeat.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+orbit:
+  enabled: true
 
 image:
   repository: zenko/backbeat
-  tag: 0.1.4
+  tag: 0.1.5
   pullPolicy: IfNotPresent
 
+api:
```

Comment:
Put this after the `consumer` definition, or move the comments on `resources` here, such that they appear at the first occurrence of `resources`.